alembic/versions/3f4e119594fe_extend_confluence_url_length.py

```diff
@@ -0,0 +1,56 @@
+# type: ignore
+# flake8: noqa
+"""extend confluence_url length
```

Comment:
`extend` -> `Extend`

Reply:
I thinks using lower case is kind of a "standard" when using `alembic` https://alembic.sqlalchemy.org/en/latest/tutorial.html#create-a-migration-script. These titles are created using:
`alembic revision -m "title"`, so the usualy are set in lower case. This happens for the rest of the migrations too :wink:.